doc/api/util.md

```diff
@@ -286,13 +286,31 @@ invoke and use the result of when inspecting the object:
 ```js
 const util = require('util');
 
-const obj = { name: 'nate' };
-obj[util.inspect.custom] = function(depth) {
-  return `{${this.name}}`;
-};
```

Comment:
I’m not sure, so this is just a suggestion, but maybe leaving the “simpler” example first would be helpful? The `Box` example is good but maybe a bit overwhelming on the first look?